modules/virt-accessing-vm-yaml-ssh.adoc

```diff
@@ -0,0 +1,135 @@
+
+.Prerequisites
+* Install the OpenShift CLI (`oc`).
+
+.Procedure
+. In the YAML file for the VM, add the label and a value for exposing the service for SSH connections. Enable the `masquerade` feature for the interface.
+
++
+.Example output
+[source,yaml]
+----
+...
+kind: VirtualMachine
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: kubevirt-hyperconverged-group
+  namespace: openshift-cnv
+  labels:
+    special: vm-ssh <1>
+  name: vm-ssh
+spec:
+  domain:
+    devices:
+      disks:
+      - disk:
+          bus: virtio
+        name: containerdisk
+      - disk:
+          bus: virtio
+        name: cloudinitdisk
+      interfaces:
+      - masquerade: {} <2>
+        name: testmasquerade <3>
+      rng: {}
+    machine:
+      type: ""
+    resources:
+      requests:
+        memory: 1024M
+  networks:
+  - name: testmasquerade
+    pod: {}
+...
+----
+<1> This label is used by the service to identify the virtual machine instances that are enabled for SSH traffic connections. The label can be any `key:value` pair that is added as a `label` to this YAML file and as a `selector` in the service YAML file.
+<2> The interface type is `masquerade`.
+<3> The name of this interface is `testmasquerade`.
+
+. Create the VM:
++
+[source,terminal]
+----
+oc create -f <path_for_the_VM_YAML_file>
+----
++
+. In the YAML file for the service, specify the service name, port number, and the target port.
+
++
+.Example output
+[source,yaml]
+----
+...
+apiVersion: v1
+kind: Service
+metadata:
+  name: svc-ssh <1>
+  namespace: openshift-cnv
+spec:
+  ports:
+      targetPort: 22 <2>
+      protocol: TCP
+  selector:
+    special: vm-ssh <3>
+  type: NodePort
+...
+----
+<1> Name of the SSH service.
+<2> The target port number for the SSH connection.
+<3> The selector name and value must match the label specified in the YAML file for the VM.
+
+. Create the service:
++
+[source,terminal]
+----
+oc create -f <path_for_the_service_YAML_file>
+----
++
+. Verify that the VM is running:
++
+[source,terminal]
+----
+oc get vm
+----
++
+.Example output
+[source,terminal]
+----
+NAME    AGE     PHASE       IP              NODENAME
+vm-ssh 6s      Running     10.244.196.152  node01
+----
+
+. Check the service to find out which port was acquired by the service:
++
+[source,terminal]
+----
+$ oc get svc
+----
++
+.Example output
+[source,terminal]
+----
+NAME            TYPE       CLUSTER-IP     EXTERNAL-IP   PORT(S)           AGE
+svc-ssh     NodePort   10.106.236.208 <none>        20022:30093/TCP   22s
+----
++
+In this example, the service acquired the port number 30093.
+
+. Log in to the VM via SSH. Use the IP address of the node and the port number that display when the `oc get svc` command is used. The following example shows the `ssh` command with the node's IP address and the port number.
++
+[source,terminal]
+----
+$ ssh username@192.168.66.101 -p  30093
```

Comment:
There's an extra space between "-p" and the port number itself (it doesn't affect the functionality, it just disturbs my eyes...).

Reply:
Updated the content.